core/commands/generate_product_repositories_command.rb

```diff
@@ -522,7 +522,7 @@ def parse_clustrix(config)
   end
 
   def parse_mdbe_ci(config)
-    raise 'Product mdbe_ci is not configured' if @env.mdbe_ci_config.nil?
+    return nil if @env.mdbe_ci_config.nil?
```

Comment:
You could just simply return an empty array here.

Reply:
I really do not like to use nil in the code of the application. We have the Result to cope with such situations.